corehq/apps/userreports/tasks.py

```diff
@@ -203,22 +212,27 @@ def queue_async_indicators():
                 now = datetime.utcnow()
                 if now > cutoff:
                     break
+                _queue_indicator(redis_client, indicator)
+
 
-                lock_key = _get_indicator_queued_lock_key(indicator)
-                lock = redis_client.lock(lock_key, timeout=60 * 60 * 6)
-                if not lock.acquire(blocking=False):
-                    continue
+def _queue_indicator(redis_client, indicator):
+    lock_key = _get_indicator_queued_lock_key(indicator)
+    lock = redis_client.lock(lock_key, timeout=60 * 60 * 24)
+    if not lock.acquire(blocking=False):
+        return
 
-                save_document.delay(indicator.id, indicator.doc_id, indicator.pillow)
+    indicator.date_queued = datetime.utcnow()
```

Comment:
I initially had this update after queueing the document, but that doesn't work when `CELERY_ALWAYS_EAGER=True` (tests) because it will process the save (which deletes the indicator) and then inserts the indicator again